index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
+            ></span>
+          </p>
+          <h5 class="card-title">Futura Dental</h5>
+          <ul>
+            <li class="badge text-bg-secondary">REACT</li>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">MySQL</li>
+          </ul>
+          <p class="card-text">
+            Created for a Dental Clinic, it shows a form to create appointments
+            and sync the data with Google Calendar, accept payments using the
+            Stripe API and a dashboard for patients to consult their treatments.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
+        </div>
+      </div>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+          <h5 class="card-title">API - Google Tag Manager</h5>
+          <ul>
+            <li class="badge text-bg-secondary">
+              Google Cloud Platform Services
+            </li>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">Google Tag Manager API</li>
+          </ul>
+          <p class="card-text">
+            ExpressJS API used for the automated deployment of Google Tag
+            Manager servers to Google Cloud Platform.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
+        </div>
+      </div>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+          <h5 class="card-title">
+            API - MySQL data to Facebook Conversions API
+          </h5>
+          <ul>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">MySQL</li>
+            <li class="badge text-bg-secondary">Facebook Conversions API</li>
+          </ul>
+          <p class="card-text">
+            ExpressJS API that imports data from a MySQL database on Azure and
+            sends Conversions API events in real time from their customer
+            interactions that cannot be tracked on the website.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="d-flex flex-column m-4 p-2 about-me">
+      <h1 class="primary py-2 my-2 text-center">About me</h1>
+      <p>
+        I can help you build a product, feature or website. If you like what you
+        see, don't hesitate to contact me.
+      </p>
+      <a href="#" class="btn btn-primary teal-button">Get my resume</a>
```

Comment:
- [ ] KIndly make sure to align correctly the about me section description and make sure to adjust the typography in order to look the same as the one on the provided Figma design :wink: 

- [ ] Kindly make sure your `Get my resume` button looks exactly like the one in the Figma design.
- [ ] Kindly make to add the background images.

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/200127560-51f2531f-60ee-47ce-9a31-26096830a98f.png)

   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/200127621-f5a4824a-5ece-4190-8454-7ed56c73f4ef.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/44822765/200138496-8493a152-3c0b-4d61-a43f-22d13dbffb1a.png)